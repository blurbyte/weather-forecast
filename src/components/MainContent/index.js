import styled from 'styled-components';

const MainContent = styled.article`
  grid-area: main;
  padding-bottom: 6rem;
`;

export default MainContent;
