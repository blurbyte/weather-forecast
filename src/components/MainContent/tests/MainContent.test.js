import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import MainContent from '../index';

test('renders correctly', () => {
  const component = renderer.create(<MainContent />);
  expect(component.toJSON()).toMatchSnapshot();
});
