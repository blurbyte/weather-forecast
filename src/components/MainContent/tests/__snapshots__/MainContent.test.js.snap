// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  grid-area: main;
  padding-bottom: 6rem;
}

<article
  className="c0"
/>
`;
